src/main/kotlin/org/rust/lang/core/psi/RustTokenType.kt

```diff
@@ -0,0 +1,10 @@
+package org.rust.lang.core.psi
+
+import com.intellij.psi.tree.IElementType
+import org.rust.lang.RustLanguage
+
+public open class RustTokenType(val debugName: String) : IElementType(debugName, RustLanguage) {}
+
+public class RustKeywordTokenType(debugName: String) : RustTokenType(debugName) {}
+
+public class RustLiteralTokenType(debugName: String) : RustTokenType(debugName) {}
```

Comment:
`public` is not necessary. It was needed because the default visibility used to be `internal`.